test/unit/entities/DutchQuote.test.ts

```diff
@@ -23,11 +23,14 @@ describe('DutchQuote', () => {
   // silent logger in tests
   const logger = Logger.createLogger({ name: 'test' });
   logger.level(Logger.FATAL);
+  process.env.ENABLE_PORTION = 'true';
```

Comment:
I think it would be better if we have a specific test that tests when ENABLE_PORTION is true and then resets the .env after. This makes all the tests have this new flow. Right?